RPIPs/RPIP-63.md

```diff
@@ -90,4 +90,4 @@ On a failed scrub check the oDAO vote to scrub a minipool. When the scrub quorum
 
 
 ## Copyright
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
\ No newline at end of file
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Also I'm not sure why github is flagging a change here :thinking:

Reply:
No idea either